### PR TITLE
feature: Add Link Component

### LINKED ISSUE #72: Crate Link component
It should be a `<a>` element.

### PR DESCRIPTION
**Pull request by nicosampler**
closes #72.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr73--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr73--safereactcomponents.review.gnosisdev.com

File: src/inputs/Link/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export const WithCustomSize = (): React.ReactElement => (
+  <Link href="#" size="lg">
+    Some text
+  </Link>
+);
+
+export const WithCustomChild = (): React.ReactElement => (
+  <Link href="#" size="lg">
+    <Icon type="alert" size="md" />
```
Comment (alongoni) [inline src/inputs/Link/index.stories.tsx:32]: The icon is a bit off. 
I think adding this it will work
```
display: flex;
align-items: center;
```

Comment (nicosampler) [reply src/inputs/Link/index.stories.tsx:32]: ok, the idea is to show that it's possible to add a child different than simple text, but I'll add a container to align it. :smile:

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr73--safereactcomponents.review.gnosisdev.com

**Review by dasanra — APPROVED** [review]

**Review by fernandomg — APPROVED** [review]

**Review by alongoni — APPROVED** [review]

**Pull request merged**